scripts/verify-replace.sh

```diff
@@ -5,7 +5,10 @@ GH_BASE_URL_KS=https://github.com/kubesaw/
 GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
 declare -a REPOS=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator" "${GH_BASE_URL_CRT}registration-service" "${GH_BASE_URL_CRT}toolchain-e2e")
 C_PATH=${PWD}
-ERRORLIST=()
+ERRORREPOLIST=()
+ERRORFILELIST=()
+GOLINTREGEX="[\s\w.\/]*:[0-9]*:[0-9]*:[\w\s)(*.\`]*"
+ERRORREGEX="Error[:]*"
```

Comment:
Could you please clarify what the `ERRORREGEX` is used for?

Reply:
I used it to get the error sentences like below and also any other error which would be there
```
make[1]: *** [vet] Error 1
make: *** [verify-replace-run] Error 1
```
but since i have changed the logic, it no longer required.